doc/install/mirrors.rst

```diff
@@ -24,7 +24,6 @@ These mirrors are available on the following locations:
 - **UK: UK**: http://uk.ceph.com
 - **US-East: US East Coast**: http://us-east.ceph.com/
 - **US-West: US West Coast**: http://us-west.ceph.com/
-- **CN: China**: http://cn.ceph.com/
```

Comment:
@zhsj  if https://mirrors.ustc.edu.cn/ceph still works and is able to offer fast access to ceph repo in mainland china. i think we can just replace "http://cn.ceph.com/" with "https://mirrors.ustc.edu.cn/ceph". what do you think?

Reply:
I don't mind, but the list here I assume is official mirror with domain *.ceph.com

Reply:
@wido thoughts?

Reply:
We can always use a non ceph.com hostname imho.

That would keep the mirror online in China, which I would prefer for our Chinese friends

Reply:
@zhsj ping?

Reply:
@tchaikov added ustc mirror domain. mirroring/MIRRORS file is not updated, so sure if it's needed for third party domain.